plugins/flutter_aepcore/ios/Classes/FlutterAEPCorePlugin.m

```diff
@@ -125,9 +138,18 @@ - (void)handleDispatchEventWithResponseCallback:(FlutterMethodCall *)call
     }
 
     [AEPMobileCore dispatch:event
-                    timeout:1
+                    timeout:timeout
             responseCallback:^(AEPEvent *_Nullable responseEvent) {
-            result([AEPEvent dictionaryFromEvent:responseEvent]);
+                if (responseEvent == nil) {
+                    FlutterError *error = [FlutterError
+                        errorWithCode:[NSString stringWithFormat:@"%ld", AEPErrorCallbackTimeout]
+                            message:@"Request timeout"
```

Comment:
Do we want to use the AdobeError code which same in the Android?
"general.callback.timeout" instead of "Request timeout"?